src/winforms/toga_winforms/widgets/table.py

```diff
@@ -0,0 +1,37 @@
+from toga.interface import Table as TableInterface
+
+from ..libs import *
+from .base import WidgetMixin      
+
+class Table(TableInterface, WidgetMixin):
+    def __init__(self, headings, id=None, style=None):
+        super(Table, self).__init__(headings, id=id, style=style)
+        self._create()
+
+    def create(self):
+        self._container = self
+        self._impl = WinForms.DataGridView()
```

Comment:
TreeView has a different implementation in Windows.
As shown in the demo, we've implemented ListView instead which has similar look and feel as the GTK version